mission/bylaws.md

```diff
@@ -0,0 +1,14 @@
+#Draft founding principles for the Foundation for Public Code
+
+## Mission
+To grow an ecosystem of public purpose open source policy and software.
+
+## Principles
+
+1. We are a non-profit and non-partisan organization.
+2. We publish under licences approved by the [Open Source Initiative](https://opensource.org/licenses) or [Creative Commons](https://creativecommons.org/licenses/) licences that do not limit commercial reuse.
+3. We are controlled by our members, who are wholly publicly controlled organisations.
+4. We strive to keep our membership as accessible as possible to potential members, regardless of size or location.
+5. We are committed to working with other organizations (commercial or non-profit) to achieve our mission.
+6. We prioritise the long-term sustainability of the ecosystem.
```

Comment:
This can be clearer.

Arnout's reply:

> Maybe: We prioritise work for active members that is in line with our long term goals.

Reply:
I feel we can add Arnout's suggestion in the sentence.

```suggestion
6. We prioritise the long-term sustainability of the ecosystem to the benefit of our members.
```

This both makes clear we change the world first but also help our members

Reply:
```suggestion
6. We prioritise the long-term sustainability of the ecosystem for the benefit of our members.
```

Reply:
We've just discussed this and come up with the following:
```suggestion
6. We prioritise long term sustainability and growth for the benefit of our members
```

Reply:
Our final decision was:

```suggestion
6. We prioritise long term sustainability and growth for the benefit of our members
```